### ISSUE TITLE
Re-enable skipped bubbletea tests

### ISSUE THREAD

**Issue by rynowak** [radius-project/radius#5762]
We had to disable some of our UI tests due to problems with the test framework provided by Bubbletea.

This item tracks re-enabling the tests. Search for the issue number to find them 👍 

[AB#8381](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/8381)

**Comment by kachawla** [comment]
Removing "triaged" label to bring this skipped test back on top.

**Comment by rynowak** [comment]
We should see if this issue has been fixed by some of the changes made to bubbletea. 

What I remember from the investigation was some inherent unreliability in the testing library provided by bubbletea (still marked experimental). I tried 3-4 different approaches and none of them worked well enough to re-enable the test. It was trivial to see locally that the tests were non-deterministic. 

That might have been addressed by now.
